engine/hid/src/hid_null.cpp

```diff
@@ -58,6 +60,7 @@ namespace dmHID
 
     void Update(HContext context)
     {
+        dmPlatform::PollEvents(context->m_Window);
```

Comment:
Old glfw automatically polls for events when doing a swapbuffers, so the hid_null doesn't respond to any mouse events (like being able to close the window).